windowsforms/README.md

```diff
@@ -21,7 +21,8 @@ If you're new to .NET Core, here are a few resources to help you understand the
 | [Matching Game](matching-game) | This sample demonstrates simple event handling and timers in a .NET Core 3 WinForms application |
 | [DataGridView Sample](datagridview) | This sample demonstrates DataGridView usage in .NET Core 3 |
 | [Graphics Sample](graphics) | This sample demonstrates using GDI+ APIs via the Graphics type in .NET Core 3 |
-| [Sudoku Sample](Sudoku) | This sample demonstrates creating a game using event handling and the Graphics type in .NET Core 3 |
+| [Sudoku Sample](Sudoku) | This sample demonstrates creating a DataGridView and the Graphics type in .NET Core 3 |
```

Comment:
I think the old description for sudoku is more accurate.
+ The link for Conway's doesn't seem to work, **perhaps** (not sure) because of spaces.

Reply:
Yes I edited the wrong line I put it back. It looks like directory names can't have spaces. I renamed the directory which caused Git to delete it and create a new one. The only changes are a directory rename plus restoring Sudoku description.